flask/db.py

```diff
@@ -74,7 +74,8 @@ def get_products():
     except BrokenPipeError as err:
         raise DatabaseConnectionError('get_products')
     except Exception as err:
-        if UNPACK_FROM_ERROR in err:
+        err_string = str(err)
```

Comment:
stringifying the error because if you don't, then the next line errors with, "unable to iterate UNPACK_FROM_ERROR in err_string becasue err_string is an object and not a string"

the `err` exception is an object at first and not a string.